components/tableNodes/NodeTable.component.tsx

```diff
@@ -61,24 +61,6 @@ export function NodeTable({ show }: NodeTableProps): JSX.Element {
     show(true, "No channels open in this node");
     return <></>;
   }
-
-  function convert(param: any): number {
-    if (param instanceof Number) {
-      return param as number;
-    }
-    return parseInt(param);
-  }
-  function check_amount_msat(param: any): string {
-    // if it is a number we return the value by appending msat
-    if (param instanceof Number) {
-      return `${param} msats`;
-    } else {
-      if (!param.includes("msat")) {
-        return `${param} msats`;
-      }
-      return param;
-    }
-  }
```

Comment:
As discussed on discord it is better keep also for the new value this check that the value is a string, at least in this period where core lightning is starting to deprecate stuff